package.json

```diff
@@ -60,6 +63,12 @@
     "karma-rollup-preprocessor": "^6.1.2",
     "karma-sauce-launcher": "^2.0.2",
     "karma-sourcemap-loader": "^0.3.7",
+    "koa": "^2.8.1",
+    "koa-esm-transform": "^1.0.0-pre.1",
+    "koa-karma-proxy": "^1.0.0-pre.4",
+    "koa-mount": "^4.0.0",
+    "koa-node-resolve": "^1.0.0-pre.6",
+    "koa-static": "^5.0.0",
```

Comment:
Can we remove `rollup` and friends from dependencies now? And maybe the karma babel plugin?